src/main/java/com/training/assignments/Operations.java

```diff
@@ -0,0 +1,98 @@
+package com.training.assignments;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import java.util.HashMap;
+
+public class Operations  {
+    private Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    public String printCompanyReviews(String companyName){
+        HashMap<String,String> company=new HashMap<>();
+        company.put("Endee engineers","good company for production of portable and online instrument of gas analyzer");
+        company.put("Aaion automation","installation of GPS tracker");
+        company.put("Thermolab","good company for production of stabilizers and incubator");
+        company.put("Shreeji solution and services","provide solutions of fire alarm system");
+        company.put("Google","Best platform where u will get solution for everything");
+        logger.info("You have entered: "+companyName+":");
+        switch(companyName)
+        {
+            case "Endee engineers":
+                logger.info("good company for production of portable and online instrument of gas analyzer");
+                break;
+            case "Aaion automation":
+                logger.info("installation of GPS tracker");
+                break;
+            case "Thermolab":
+                logger.info("good company for production of stabilizers and incubator");
+                break;
+            case "Shreeji solution and services":
+                logger.info("provide solutions of fire alarm system");
+                break;
+            case "Google":
+                logger.info("Best platform where u will get solution for everything");
+                break;
+            default:
+                logger.info("All companies are good");
+        }
+        return company.get(companyName);
+    }
+
+    public String conditionCheckUsingIf(String people){
+        HashMap<String,String> behaviour=new HashMap();
+        behaviour.put("good","go to heaven");
+        behaviour.put("very good","go to heaven with dog");
+        behaviour.put("Bad","go to hell");
+        behaviour.put("Very Bad","go to hell");
+
+        if(people == "good") {
+            logger.info("go to heaven");
+        } else if(people == "very good"){
+            logger.info("go to heaven with dog");
+        } else if(people == "Bad"){
+            logger.info("go to hell");
+        }else if(people == "Very Bad"){
+            logger.info("go to hell");
+        }
```

Comment:
are you just using if else to print to logger?
Can you review your question - @PranavHanwante